ui/app/AppLayouts/Wallet/controls/ManageTokensGroupDelegate.qml

```diff
@@ -0,0 +1,148 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+DropArea {
+    id: root
+    objectName: "manageTokensGroupDelegate-%1".arg(index)
+
+    property int visualIndex: index
+    property var controller
+    property bool communityGroupsExpanded
+    property var dragParent
+    property alias dragEnabled: groupedCommunityTokenDelegate.dragEnabled
+    property bool isCollectible
+
+    readonly property string communityId: model.communityId
+    readonly property int childCount: model.enabledNetworkBalance // NB using "balance" as "count" in m_communityTokenGroupsModel
```

Comment:
Could you explain what do you refer in this comment? I'm not sure if I'm understanding what the `childCount` is expected to represent.

Reply:
The number of child delegates (ie the tokens belonging to this group); will add a TODO to the model to add an extra role

Reply:
Ok, but not the total balance? just the number of delegates inside the group, right?

Reply:
Yup, correct